evm/chains/base.py

```diff
@@ -427,13 +456,13 @@ def get_block_by_hash(self, block_hash):
     #
     @classmethod
     def from_genesis(cls,
-                     chaindb,
-                     genesis_params,
-                     genesis_state=None):
+                     chaindb: BaseChainDB,
+                     genesis_params: Dict,
+                     genesis_state: Dict[Address, Dict]=None) -> 'BaseChain':
```

Comment:
Same here, can we get rid of the *implicit* `Any` for the `Dict` by *explicitly* setting the types.